plugins/clnrest/utilities/generate_certs.py

```diff
@@ -7,7 +7,7 @@
 import datetime
 
 
-def generate_ca_cert(certs_path):
+def generate_ca_cert(rest_host, certs_path):
```

Comment:
this is a really good catch. Very good job!

Reply:
Thanks!